app/models/saved_claim/education_career_counseling_claim.rb

```diff
@@ -33,4 +33,19 @@ def process_attachments!
   def business_line
     'EDU'
   end
+
+  def send_failure_email
```

Comment:
Same as above, please
add docblock and some reference to how you might approach this in a refactor if you had time.

Reply:
done!